container-host-files/etc/scf/config/scripts/configure-az.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+set -o nounset ;# trap undefined variables
+set -o errexit ;# abort on error
+
 if test -z "${AZ_LABEL_NAME}"
```

Comment:
Wouldn't this be an issue now that we have `nounset`? Or do we always provide an `AZ_LABEL_NAME`, that just happens to be empty sometimes?

Reply:
No, the variable is always set, just empty sometimes.

Reply:
Removed the `|| true` with commit 4b739f7c4af0a82bd6ee3bd6348edb058a7a0849